src/styles/stars.scss

```diff
@@ -0,0 +1,14 @@
+.stars {
+  display: flex;
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background: url('../images/star-active.svg') no-repeat center center/contain;
```

Comment:
Consider using SCSS variables for the URL paths to the images to ensure consistency and ease of updates.